extract_endpoint/src/azure_utils.py

```diff
@@ -0,0 +1,35 @@
+import enum
+import io
+import typing
+from azure.storage import blob
+
+
+class StorageCoordinates(typing.NamedTuple):
+    account: str
+    key: str
+    container: str
+    domain: typing.Optional[str]
```

Comment:
Does this need to be optional?

Reply:
Yes, if you set the domain to `None` then it connects to the real Azure.

Reply:
👍